src/redux/data.ts

```diff
@@ -403,42 +866,87 @@ const checkForConfirmedSavingsActions = transactionsData => dispatch => {
   }
 };
 
-const checkForUpdatedNonce = transactionData => dispatch => {
+/**
+ * Checks to see if a network's nonce should be incremented for an acount
+ * based on incoming transaction data, and if so, updates state.
+ *
+ * @param transactionData Incoming transaction data.
+ */
+const checkForUpdatedNonce = (transactionData: ZerionTransaction[]) => (
+  dispatch: ThunkDispatch<AppState, unknown, never>
+) => {
   const txSortedByDescendingNonce = transactionData.sort(
+    // @ts-expect-error TypeScript prevents this subtraction since `n1` or `n2`
+    // may be null, but subtracting null and a number is possible.
     ({ nonce: n1 }, { nonce: n2 }) => n2 - n1
```

Comment:
Would it be better to explicitly fall back to zero?

```suggestion
    ({ nonce: n1 }, { nonce: n2 }) => (n2 ?? 0) - (n1 ?? 0)
```

Reply:
Ah yes, that makes sense!